modules/core/src/main/java/org/apache/ignite/internal/processors/bulkload/BulkLoadFormat.java

```diff
@@ -21,14 +21,8 @@
 /**
  * A superclass and a factory for bulk load format options.
  */
-public abstract class BulkLoadFormat {
+public interface BulkLoadFormat {
     /** The default input charset. */
-    public static final Charset DEFAULT_INPUT_CHARSET = Charset.forName("UTF-8");
+    Charset DEFAULT_INPUT_CHARSET = Charset.forName("UTF-8");
```

Comment:
```suggestion
Charset DFLT_INPUT_CHARSET = StandardCharsets.UTF_8;
```

abbreviation should be used and chraset we can obtain more safety.